aspnetcore/fundamentals/middleware/index.md

```diff
@@ -533,6 +533,7 @@ ASP.NET Core ships with the following middleware components. The *Order* column
 | [HTTP Strict Transport Security (HSTS)](xref:security/enforcing-ssl#http-strict-transport-security-protocol-hsts) | Security enhancement middleware that adds a special response header. | Before responses are sent and after components that modify requests. Examples: Forwarded Headers, URL Rewriting. |
 | [MVC](xref:mvc/overview) | Processes requests with MVC/Razor Pages. | Terminal if a request matches a route. |
 | [OWIN](xref:fundamentals/owin) | Interop with OWIN-based apps, servers, and middleware. | Terminal if the OWIN Middleware fully processes the request. |
+| [Output Caching](xref:performance/caching/output) | Provides support for caching responses based on configuration. | Before components that require caching. `UseRouting` must come before `UseOutputCaching`.|
```

Comment:
I think this will add output caching to the < 6.0 version of the table. The table around line 230 is >= 6.0